src/main/webapp/app/entities/clinical-trials-gov-condition/clinical-trials-gov-condition-detail.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { connect } from 'app/shared/util/typed-inject';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { Button, Row, Col } from 'reactstrap';
+import {} from 'react-jhipster';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { IRootStore } from 'app/stores';
+import { APP_DATE_FORMAT, APP_LOCAL_DATE_FORMAT } from 'app/config/constants';
+import { getCancerTypeName } from 'app/shared/util/utils';
+export interface IClinicalTrialsGovConditionDetailProps extends StoreProps, RouteComponentProps<{ id: string }> {}
+
+export const ClinicalTrialsGovConditionDetail = (props: IClinicalTrialsGovConditionDetailProps) => {
+  useEffect(() => {
+    props.getEntity(props.match.params.id);
+  }, []);
+
+  const clinicalTrialsGovConditionEntity = props.clinicalTrialsGovConditionEntity;
+  return (
+    <Row>
+      <Col md="8">
+        <h2 data-cy="clinicalTrialsGovConditionDetailsHeading">CT.gov Condition</h2>
```

Comment:
Perhaps it is better to use full "ClinicalTrials.gov" in the heading here

Reply:
I want to keep it short. CT.gov is a well perceived usage.